scripts/mixins/families/aern.lua

```diff
@@ -20,21 +20,51 @@ g_mixins.families.aern = function(aernMob)
                     reraises = 1
                 end
             end
+            mob:setMobMod(xi.mobMod.NO_DROPS, 0) -- Drops on if Not reraising
             if curr_reraise < reraises then
+                mob:setMobMod(xi.mobMod.NO_DROPS, 1)  -- Drops off if reraising
                 local dropid = mob:getDropID()
-                mob:setDropID(0)
+                local owner = nil
                 local target = mob:getTarget()
-                if target then killer = target end
```

Comment:
I believe the likely intend was to use target if it existed and fallback to killer if it didn't, but the originator of this line overwrite killer with target rather than overwriting target with killer (backwards assignment). I am not sure what retail actually does if the target is died just before the aern performs its self-raise. If it does NOT go for another target, such a fallback isn't needed.

Reply:
looking in the history it didn't fall back, just gave up if no target was found.

Reply:
It aggros other party members. IE the reason for my getAlliance() section now